filechanges/etc/samba/smb.conf

```diff
@@ -86,7 +86,7 @@ follow symlinks = yes
     directory mask = 0777
 
 #[Webclient]
-#    path = /opt/defaultwebclient
+#    path = /opt/webclient
```

Comment:
It could also be that @woutervanwijk shared these folders to allow for easy modification and testing while doing development on another machine?